src/validation/index.ts

```diff
@@ -1,7 +1,7 @@
 import { GenericValidator } from "./generic";
-import { blankFieldValidations } from "./enter.your.details.validation";
+import { formValidation } from "./enter.your.details.validation";
 
 export default  {
```

Comment:
This file seems a bit pointless, is there a reason for it, why can't re just import the validation into the routers directly?

Reply:
I thought we might have other ones but since it is a small project you are right